SOURCES/netdata.conf.headless

```diff
@@ -0,0 +1,40 @@
+# netdata configuration
```

Comment:
Shipping modified copies of the upstream config makes further upgrades error-prone, if someone not aware of this just imports a newer source package touching original config.
I'm not sure we can make sure this is handled properly with just a textual note, so a hard-failure when this happen could be useful. That could be shipping those changes as patches against the original config, declared as `Source` file in the spec, and generating the headless and ui variants by calling `patch` in the `%build` step.